code/standards/checkstyle.xml

```diff
@@ -13,13 +14,16 @@
       <property name="allowUndeclaredRTE" value="true"/>
       <property name="allowMissingParamTags" value="true"/>
       <property name="allowMissingReturnTag" value="true"/>
+      <property name="allowMissingThrowsTags" value="true" />
       <property name="allowMissingPropertyJavadoc" value="true"/>
     </module>
     <module name="JavadocType"/>
     <module name="JavadocVariable">
       <property name="scope" value="public"/>
     </module>
-    <module name="ConstantName"/>
+    <module name="ConstantName">
+      <property name="applyToPrivate" value="false" />
```

Comment:
A number of classes already violate this and I don't want to introduce a motive for churn. In addition we currently have tens of thousands of warnings: Ideally we could cut that number into one that can be meaningfully audited.